addons/mail/static/src/js/services/mail_notification_manager.js

```diff
@@ -361,6 +361,8 @@ MailManager.include({
             this._handlePartnerMailFailureNotification(data);
         } else if (data.type === 'user_connection') {
             this._handlePartnerUserConnectionNotification(data);
+        } else if (data.type === 'simple_notification') {
+            data.warning ? this.do_warn(data.title, data.message, data.sticky) : this.do_notify(data.title, data.message, data.sticky);
```

Comment:
`warning` not used yet. Could be introduced later when it's used.

Reply:
I used in a previous version but I found it very agressive, I will run it with a PO to see if they want the warning or not for my usecase update the code if needed.